.github/workflows/gnulinux.yml

```diff
@@ -96,7 +96,7 @@ jobs:
           echo "compiler=$toolchain_compiler" >> $GITHUB_ENV
           echo "compiler_version=$toolchain_version" >> $GITHUB_ENV
       - name: Setup Rust toolchain
-        uses: dtolnay/rust-toolchain@v1
+        uses: dtolnay/rust-toolchain@nightly
```

Comment:
mmm sure of that ? nightly being a moving tag this is not reproducible :-/ (even v1 is, but ensuring retro-compat)

Reply:
from what i know, github runner image got rust pre installed **and** we generate a rust-toolchain.toml file which installed the given target/channel while building.

This might not be necessary anymore.

Reply:
> https://docs.github.com/en/actions/use-cases-and-examples/building-and-testing/building-and-testing-rust#introduction